CONTRIBUTING.md

```diff
@@ -11,6 +11,45 @@ through the issue tracker with your idea so that we can help out and possibly
 guide you. Coordinating up front makes it much easier to avoid frustration later
 on.
 
+# Development
```

Comment:
This shouldn't be in this PR?

Reply:
Argh! :-(

Probably I have rebased against the wrong branch at some point.

Reply:
Git can be hard!

Reply:
Even more so when you have to switch between different git workflows!